modules/terraform-zscc-gwlb-aws/main.tf

```diff
@@ -2,11 +2,12 @@
 # Configure target group
 ################################################################################
 resource "aws_lb_target_group" "gwlb_target_group" {
-  name        = "${var.name_prefix}-cc-target-${var.resource_tag}"
-  port        = 6081
-  protocol    = "GENEVE"
-  vpc_id      = var.vpc_id
-  target_type = var.asg_enabled == true ? "instance" : "ip"
+  name                 = var.target_group_name
+  port                 = 6081
+  protocol             = "GENEVE"
+  vpc_id               = var.vpc_id
+  target_type          = var.asg_enabled == true ? "instance" : "ip"
+  deregistration_delay = var.deregistration_delay
```

Comment:
@anichari 

GWLB (ASG and regular deployment changes)
- deregistration_delay set to 0 : CHANGE

Reply:
@jmolnar-zscaler Should this be a variable or hardcoded to 0?